.github/workflows/unit-tests.yml

```diff
@@ -130,7 +130,9 @@ jobs:
           [[ "${{ matrix.mode }}" = *-deps ]] && mv composer.json.phpunit composer.json || true
 
           if [[ "${{ matrix.mode }}" = low-deps ]]; then
-            echo SYMFONY_PHPUNIT_REQUIRE="nikic/php-parser:^4.18" >> $GITHUB_ENV
+            echo SYMFONY_PHPUNIT_REQUIRE="nikic/php-parser:'^4.18|^5.0'" >> $GITHUB_ENV
+          else
+            echo SYMFONY_PHPUNIT_REQUIRE="sebastian/comparator:'<6.2.0'" >> $GITHUB_ENV
```

Comment:
`sebastian/comparator` was released this afternoon and has a bug (accessing an uninitialized string offset) that lets some of our tests fail (see sebastianbergmann/comparator#118).

Reply:
bugfix released